hazelcast/src/main/java/com/hazelcast/jet/impl/pipeline/transform/ProcessorTransform.java

```diff
@@ -42,6 +42,7 @@
 
 public class ProcessorTransform extends AbstractTransform {
     public static final int NON_COOPERATIVE_DEFAULT_LOCAL_PARALLELISM = 2;
+    private static final long serialVersionUID = 1L;
```

Comment:
It's a minor style thing but I would
- put the `serialVersionUID` field always first, so it's more visible if anyone is making changes
- separate it with empty line before & after

Reply:
Yes, I wanted to put it first but unfortunately, checkstyle has a different opinion
```
[ERROR] /Users/ldziedziul/projects/hazelcast/hazelcast/hazelcast/src/main/java/com/hazelcast/jet/impl/pipeline/transform/ProcessorTransform.java:46:5: Variable access definition in wrong order. [DeclarationOrder]
```
Empty lines added here: https://github.com/hazelcast/hazelcast/pull/19534/commits/ddfbf33b32e0c0639dd8f05b925b00f4eed1e4e1